cmd/dev_common.go

```diff
@@ -291,9 +288,14 @@ func commonCmd(config *devCommonConfig, mode string) error {
 			// 'signal: interrupt'
 			// TODO presumably you can query the error itself
 			error := fmt.Sprintf("%s", err)
+			Debug.log("CLI exit error is:  ", error)
 			//Linux and Windows return a different error on Ctrl-C
-			if error == "signal: interrupt" || error == "exit status 2" {
-				Info.log("Closing down, development environment was interrupted.")
+			if error == "signal: interrupt" || error == "signal: terminated" || error == "exit status 2" {
+				Info.log("Closing down, development environment was interrupted will now sleep 10 seconds.")
```

Comment:
I don't like saying `will now sleep 10 seconds`.
Lets just leave it at
`Closing down, development environment was interrupted.`

Reply:
changed